### PR TITLE
Shared todo example [WIP]

### PR DESCRIPTION
**Pull request by acekyd**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 👷 Example Application
- [ ] 🧑‍💻 Code Snippet
- [ ] 🎨 Design
- [ ] 📖 Content
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

## Related Tickets & Documents

Resolves #

## Added code snippets?
- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### No tests? Add a note

Building an example app for a tutorial.

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

Documentation in Progress

## [optional] Are there any post-deployment tasks we need to perform?

- [x] Configure and Deploy to CodeSandbox
- [ ] Review code for best practices and improvements
- [ ] Write out tutorial
- [ ] Update readme, merge PR and publish tutorial article.

## [optional] What gif best describes this PR or how it makes you feel?

Cooking

### CODE REVIEW

File: examples/tutorials/shared-todo-completed/pages/index.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+        </div>
+        <div v-if="!showForm" class="text-center">
+            <button @click="showForm = true" class="btn btn-primary">Create Shared Todo List</button>
+        </div>
+    </div>
+</template>
+
+<script setup>
+import { ref } from 'vue';
+import { Web5 } from '@web5/api/browser';
```
Comment (angiejones) [inline examples/tutorials/shared-todo-completed/pages/index.vue:50]: did you try without `/browser`? That *shouldn't* be a requirement but I know there were some issues

**Review by angiejones — APPROVED** [review]

File: examples/tutorials/shared-todo-completed/pages/index.vue (hunk 1 above)
Comment (acekyd) [reply examples/tutorials/shared-todo-completed/pages/index.vue:50]: It was causing a different error previously but it does seem to work now.

**Comment by acekyd** [comment]
Cleaned this up and ready to merge once a decision is made regarding the socket security issue.

**Comment by angiejones** [comment]
@SocketSecurity ignore-all

**Comment by angiejones** [comment]
@acekyd looks like there's some broken links:

> 6:38:44 AM: [ERROR] Unable to build website for locale en.
> 6:38:44 AM: [ERROR] Error: Docusaurus found broken links!
> 6:38:44 AM: 
> 6:38:44 AM: Please check the pages of your site in the list below, and make sure you don"t reference any path that does not exist.
> Note: it"s possible to ignore broken links with the "onBrokenLinks" Docusaurus configuration, and let the build pass.
> 6:38:44 AM: Exhaustive list of all broken links found:
> 6:38:44 AM: 
> 6:38:44 AM: - On source page path = /docs/web5/build/apps/collaborative-todo-app:
> 6:38:44 AM:    -> linking to /todo-app-tutorial
> 6:38:44 AM:    -> linking to "docs/web5/learn/protocols" (resolved as: /docs/web5/build/apps/"docs/web5/learn/protocols")

**Comment by acekyd** [comment]
All good to go now! @angiejones

**Pull request merged**